docs/team/ytingtan.md

```diff
@@ -10,17 +10,74 @@ CLInkedIn operates mainly using a Command Line Interface (CLI), but still has a
+        2. Editing the rating of a candidate `edit 2 rate/9` edits `Rating` of the 2nd candidate in CLInkedIn to `9`.
+        3. Deleting rating of a candidate `deleterate 1` deletes rating of the 1st candidate in CLInkedIn. 
+    * Implementation specifications: 
+      * Rating is internally implemented as an Integer. 
+      * Rating is implemented as a non-compulsory attribute under a `Person` object. Thus, a `Rating` object class is created and added to `Person` as a field.
+      * For candidates with no rating, it is internally represented as having a `Rating` of `0`. However, it is not shown in the external interface.
+
+* **New Feature**: Sort 
+    * What it does: 
+      * Order all candidates from the highest rating to the lowest rating.
+      * Recruiters can view all candidates in order of desirability.
+      * Example: `sort` will return the list of candidates sorted from the highest to the lowest rating. 
+      * The candidates with no rating will be placed at the bottom, as there is not enough information to compare them with the rest of the applicant pool.
+    * Implementation specifications: 
+      * A `Comparator<Person>` is created to compare `Rating` between 2 candidates. 
+      * All candidates go through the comparator, and a sorted list is obtained. 
 
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=ytingtan&breakdown=true)
 
 
 * **Project management**:
-
+  * PRs reviewed and merged: 
+    * [\#40] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/40)
+    * [\#42] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/42)
+    * [\#52] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/52)
+    * [\#53] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/53)
+    * [\#66] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/66)
+    * [\#68] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/68)
+    * [\#71] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/71)
+    * [\#77] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/77)
+    * [\#83] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/83)
+    * [\#85] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/85)
+    * [\#86] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/86)
+    * [\#103] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/103)
+    * [\#105] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/105)
+    * [\#109] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/109)
+    * [\#138] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/138)
+    * [\#141] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/141)
+    * [\#142] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/142)
+    * [\#205] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/205)
+    * [\#207] (https://github.com/AY2223S1-CS2103T-T13-3/tp/pull/207)
```

Comment:
Can consider formatting in a single line just in case! Tried converting my own PPP to pdf to test and seems like yours might exceed the two-page limit 🥲

Reply:
thanks for the tip! didn't know about the limit!